.gitignore

```diff
@@ -2,4 +2,5 @@
 /bin/php-cs-fixer
 /bin/phpspec
 /vendor
+composer.lock
```

Comment:
I agree with ignoring composer.lock, but then you should delete such file (instead of updating it)

Reply:
Never mind. I loaded diff and seen that is actually a delete